frontend/src/components/buttons/confirmationButton/ConfirmationButton.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
-import {Button} from "@mui/material";
+import {Button, ThemeProvider} from "@mui/material";
 import {ConfirmationButtonProps} from "./props";
 import {Buttons} from "../../../common/constansts/buttons";
+import {buttonTheme} from "../../../common/themes/button";
 
 const ConfirmationButton: React.FC<ConfirmationButtonProps> = ({handleClick, buttonName, isDisabled, className}) => {
     return (
-        <Button
-            onClick={handleClick}
-            variant="contained"
-            className={`button ${className}`}
-            disabled={isDisabled}
-        >
-            {buttonName ? buttonName : Buttons.CONFIRM}
-        </Button>
+        <ThemeProvider theme={buttonTheme}>
+            <Button
+                onClick={handleClick}
+                variant="contained"
+                color="blue"
+                className={`button ${className}`}
+                disabled={isDisabled}
+            >
+                {buttonName ? buttonName : Buttons.CONFIRM}
+            </Button>
```

Comment:
It would be best if you don't need to touch this and cancel button components at all. It means you rewrite primary and secondary color styles and they automatically appplies for all components in your project where you have already used these styles

Reply:
But i already don't have to touch these components when i change primary or secondary color? i dont understand what you mean

Reply:
add changes to styles such way that you don't need to change this file at all